### ISSUE TITLE
Possible mismatch in data type in bad pixel map test.

### ISSUE THREAD

**Issue by ecady-jpl** [roman-corgi/corgidrp#272]
**Describe the bug**
At [this location](https://github.com/roman-corgi/corgidrp/blob/v1.1.2/tests/test_badpixelmap.py#L75) in the badpixelmap test code, `np.unpackbits` is used to separate the bits in the bad pixel map.  However, the DQ element [is a 64-bit integer](https://github.com/roman-corgi/corgidrp/issues/267#issuecomment-2549317604), but `np.unpackbits` [only works on uint8s](https://numpy.org/doc/stable/reference/generated/numpy.unpackbits.html).  Any bits beyond the first 8 will be dropped, and a DeprecationWarning will be raised if any are set.

This is not currently a breaking bug as the corgidrp implementation document only assigns values up to bit 7, and higher ones are TBD.  But when that TBD is defined, this is likely to appear when running the test suite, and may cause test failures.

**To Reproduce**
```
>>> import numpy as np
>>> np.unpackbits(np.array([[0], [1], [255]], dtype=np.uint8), axis=1)
array([[0, 0, 0, 0, 0, 0, 0, 0],
       [0, 0, 0, 0, 0, 0, 0, 1],
       [1, 1, 1, 1, 1, 1, 1, 1]], dtype=uint8)
>>> np.unpackbits(np.array([[0], [1], [255], [256]], dtype=np.uint8), axis=1)
<stdin>:1: DeprecationWarning: NumPy will stop allowing conversion of out-of-bound Python integers to integer arrays.  The conversion of 256 to uint8 will fail in the future.
For the old behavior, usually:
    np.array(value).astype(dtype)
will give the desired result (the cast overflows).
array([[0, 0, 0, 0, 0, 0, 0, 0],
       [0, 0, 0, 0, 0, 0, 0, 1],
       [1, 1, 1, 1, 1, 1, 1, 1],
       [0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
```

**Expected behavior**
A different tool needs to be used to separate the bits in the DQ bit mask

**corgidrp version:**
 - v1.1.2

**Comment by semaphoreP** [comment]
I think this is now addressed in #317. We use a custom function to unpack bits that should support > 8 bits.

**Comment by maxwellmb** [comment]
@alexisyslau, @semaphoreP does this need to be re-opened?

**Comment by semaphoreP** [comment]
No, this problem is solved. We just created possible another problem (although no tests have failed, so I don't think it's affecting anything right now).

**Comment by alexisyslau** [comment]
Exactly, I am writing unit tests for the packing and unpacking. I might also create a mapping function just to make it easier to know which flag corresponds to which bits (using dictionary), if this helps.
